ucp/_libs/ucx_api.pyx

```diff
@@ -312,7 +312,7 @@ cdef class UCXWorker(UCXObject):
         cdef ucs_status_t status = ucp_ep_create(self._handle, &params, &ucp_ep)
         c_util_get_ucp_ep_params_free(&params)
         assert_ucs_status(status)
-        return ucx_ep_create(ucp_ep, self)
+        return UCXEndpoint(self, <uintptr_t><void*> ucp_ep)
```

Comment:
Is this necessary? Do you need to cast to `void*` and then cast it again to `uintptr_t` ?

Reply:
IOW, isn't it possible to directly cast once as `<uintptr_t> ucp_ep`?

Reply:
Fixed